.github/workflows/tests.yaml

```diff
@@ -12,16 +12,16 @@ jobs:
     strategy:
       matrix:
         os: ["ubuntu-latest", "macos-latest", "windows-latest"]
-        python-version: ['3.8', '3.9', '3.10', '3.11']
+        python-version: ['3.9', '3.10', '3.11']
     steps:
       - uses: actions/checkout@v3
       - name: Set up Python
         uses: actions/setup-python@v4.3.0
         with:
           python-version: ${{ matrix.python-version }}
-      - name: Install dependencies
+      - name: Upgrade pip and install test dependencies
         run: |
-          pip install -e .
+          pip install --upgrade pip
```

Comment:
Just curious, why remove the `pip install -e .`? I would have thought it was needed?

Reply:
Its because I now understand `pip` a little better and `pip install -e .[tests]` installs not just the tests but the optional dependencies defined under `tests` in addition to the the `dependencies`.